### PR TITLE
Review app2version

### LINKED ISSUE #106: User Review
As an app user I want to be able to rate the application, so that I can share my feedback with the developers.

Acceptance criteria:
- [x] The app has a start rate bar
- [ ] The app allows to submit the review only once.
- [ ] The app stores user reviews in database.

### PR DESCRIPTION
**Pull request by gusciuag**
⛓ **Description:**

- Closes #106

_New version of review option, improved layout and code logic regarding the user review part of the app, added screen in the drawer where users reviews are displayed._

🛠 **Type of change:**
- [x] The app allows to submit the review only once.
- [x] The app stores user reviews in database.
- [x] The information is displayed on separate screen stored in the drawer.

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

File: GreenGarbageApp/app/src/main/res/layout/fragment_leaderboard.xml [hunk 1]
```diff
@@ -64,6 +64,17 @@
             android:textColor="@color/white"
             android:text="Points"
             android:textSize="24sp"/>
+        <TextView
```
Comment (Toerngren) [inline GreenGarbageApp/app/src/main/res/layout/fragment_leaderboard.xml:67]: Why is the review result on the scoreboard? What has the review with the game result to do?  Didn't we discussed to add the result of all reviews to the drawer?

**Comment by Toerngren** [comment]
Fixed merge conflict.

**Review by Toerngren — APPROVED** [review]

**Comment by Toerngren** [comment]
@Adrianmeda, re-check these please: 

- [ ] The app allows to submit the review only once.
- [ ] The app stores user reviews in database.
- [ ] The information is displayed on separate screen stored in the drawer

**Review by Adrianmeda — APPROVED** [review]
Worked good when I had driven the car, did not save when I tried to directly do a review without driving the car. I guess it does not really matter because one has to try the app to do a review!

**Comment by Toerngren** [comment]
> Worked good when I had driven the car, did not save when I tried to directly do a review without driving the car. I guess it does not really matter because one has to try the app to do a review!

Thank you @Adrianmeda, this has been discussed (Agata and me), you need to play the game to review it. As its location now on the save fragment, I think that makes sense.

**Pull request merged**